css/style.css

```diff
@@ -15,5 +26,240 @@ body {
+     list-style: none;
+     display:flex;
+     justify-content: space-between;
+ }
+
+ .nav-mobile {
+    width:100%;
+    padding: 0;
+    list-style: none;
+ }
+
+ .nav-mobile li {
+    padding: 5px 0;
+ }
+
+ .navigation a, .nav-mobile a {
+    text-decoration: none;
+    color:var(--grey-dark);
+}
 
+.strong {
+    color:var(--grey-darker);
+    font-weight: 500;
+}
+
+.navigation a:hover, .nav-mobile a:hover{
+    color: var(--orange-dark);
+} 
+
+.hamburger{
+    display:none;
+}
+
+.hero {
+    background-image: url("../img/first-background.jpg");
+    background-size: cover;
+    height: 60vh;
+    display:flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    color:var(--white);
+    text-align: center;
+    padding: 0 10%;
+}
+
+button {
+    background-color:var(--orange-dark);
+    padding: 15px 35px;
+    border-radius: 15px;
+    margin-top:30px;
+    border:0;
+    color:var(--white)
+}
+
+button:hover{
+    background-color:var(--grey-darker);
+    color:var(--orange-light);
+}
+
+.hero h1 {
+    font-weight: 400;
+    font-size:56px;
+    margin-bottom:0px;
+}
+
+.hero h2 {
+    font-weight: 400;
+    font-size:31px;
+    margin-bottom:10px;
+}
+
+.main {
+    padding:30px 7%;
+}
+
+.main h3 {
+    text-align:center;
+    font-size: 3rem;
+    font-weight: 400;
+}
+
+.feature-container {
+    display:flex;
+    justify-content: space-around;
+    padding-bottom:2rem;
+}
+
+.feature {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-size: 1.5rem;
+    text-align:center;
+    margin:0.5rem;
+}
+.feature-image {
+    width: 10vw;
+    padding-bottom:1rem;
+}
+.promo {
+    margin:40px 0;
+    display: flex;
+    width:100%;
+    min-height:300px;
+    background-color: var(--orange-light);
+}
+.promo-img {
+    width:40%;
+    overflow: hidden;
+}
+.promo-img img{
+    height:100%;    
+}
+.quote-container {
+    display: flex;
+    width:60%;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding:0;
+    padding-bottom:20px;
+}
+.quote {
+    quotes: "“" "”" "‘" "’";
+}
+.quote ::before {
+    content: open-quote;
+    color:var(--orange-dark);
+    font-size:4rem;
+    padding-right: 0.3rem;
+}
+.quote ::after {
+    content: close-quote;
+    color:var(--orange-dark);
+    font-size:4rem;
+    padding-left: 0.3rem;
+    line-height: 0.7;
+}
+.quote-container blockquote {
+    text-emphasis: 1;
+    font-size:40px;
+    text-align: center;
+    margin:20px 20px 0 20px;
+}
+
+footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-top:solid 1px var(--grey-dark);
+}
+
+
+.social-container {
+    display:flex;
+}
+
+.social-container > a {
+    display:flex;
+    padding:5px;
+    margin: 5px;
+    justify-content: center;
+    width: 45px;
+    height:45px;
+    border: solid 1px var(--grey-dark);
+    border-radius:50%;
+    align-items: center;
+    transition: 0.5s;
+}
```

Comment:
The social buttons are larger than in the reference design, please reduce the size. Also, the circle outline around the images should be slightly lighter according to the design.